slaves/linux-cross/Dockerfile

```diff
@@ -38,6 +36,30 @@ RUN pip install buildbot-slave
 RUN groupadd -r rustbuild && useradd -r -g rustbuild rustbuild
 RUN mkdir /buildslave && chown rustbuild:rustbuild /buildslave
 
+# Install arm cross compiler
+WORKDIR /build
+COPY linux-cross/build_arm_toolchain_root.sh /build/
+RUN /bin/bash build_arm_toolchain_root.sh && chown rustbuild:rustbuild /build
+RUN mkdir /x-tools && chown rustbuild:rustbuild /x-tools
```

Comment:
do we need to `chown`? rustbuild shouldn't want to modify anything in these directories, right? It's fine to just run all the builds as `root` I think.